docs/CONTRIBUTING.md

```diff
@@ -23,6 +25,21 @@ Once you have cloned the JsDbg repository, please follow the steps below to buil
       * Common build errors and fixes:
          * Missing Windows 10 SDK error - install the correct SDK version (should be specified in the error message) and rebuild the solution.
 
+#### Linux
+
+1. Get the [x64 .NET Core SDK binary](https://dotnet.microsoft.com/download) (click the x64 link in the left column in the table at the bottom). If you extract this into a different directory than `~/dotnet`, adjust the instructions below (and if you want to use `create_package.sh`, adjust the path to dotnet in this script)
+2. To build the server, run `~/dotnet/dotnet publish -c Release -r linux-x64` from the `server/JsDbg.Gdb` directory.
+3. The command will tell you where the output is; it should be in `bin/Release/netcoreapp2.1/linux-x64/publish` under the JsDbg.Gdb directory.
+4. Add the following code to your `~/.gdbinit`:
+```
+python
+import sys
+sys.path.insert(0, "<your home directory>/JsDbg/server/JsDbg.Gdb")
+import JsDbg
+end
+```
+5. If you cloned into `~/JsDbg` it should find the binary and extensions directory automatically; otherwise you will have to edit `JsDbg.py` and update the path.
```

Comment:
I think "update the path" is a little unclear here. Could you make this part a little more explicit?

Reply:
Done